app/models/account.js

```diff
@@ -4,7 +4,7 @@ import { alias } from '@ember/object/computed';
 // TODO: octane-refactor
 // eslint-disable-next-line ember/no-classic-classes
 export default Model.extend({
-  user: belongsTo('person'),
```

Comment:
person model is a `person:Person`, while user is a `foaf:Person`.
person has a different first-name predicate `persoon:gebruikteVoornaam`.
Need to investigate what the consequences are from this small change

Reply:
@ValenberghsSven No impact, since the relation is correctly defined in the backend models. It's just the frontend where the relationship is wrong. See also https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1216#discussion_r819454975

Reply:
I think the reason we did this is because the email and phone come from acmidm,  but if we use the `user` model, you technically override that data with every save..

Reply:
I completely missed this change/conversation in PR1216
Will be responding in that thread instead